README.md

```diff
@@ -8,9 +8,15 @@ It is used in the [Origami registry](http://registry.origami.ft.com), the [image
 
 __o-techdocs__ has no silent mode, so the provided CSS classes beginning with `o-techdocs` must be used. See the demo page for examples.
 
+## Upgrading from v5.x.x to v6.x.x
+
+v6 of `o-techdocs` removes a dependency on `o-header` v4. To upgrade, you'll need to do a find-replace on instances of `o-header` for `o-techdocs-header`.
+
+In moving `o-header` back into `o-techdocs-header` it's possible that we've failed to migrate some features across. If, after updating to v6 and renaming `o-header` to `o-techdocs-header`, you find that things aren't working, please raise an issue.
+
 ## Upgrading from v4.x.x to v5.x.x
 
-o-techdocs v5 relies on o-header v4 and o-grid v4.
+o-techdocs v5 relies on o-techdocs-header v4 and o-grid v4.
```

Comment:
Should this be upgrading to from `v4.x.x to v6.x.x` if you're updating this line?